LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Should also be changed in the `composer.json`

Reply:
done